src/client/components/searchBar.mli

```diff
@@ -24,5 +25,8 @@ val make :
 
     - [on_enter] is a function that triggers when the user presses Enter.
 
+    - [autofocus] is a boolean that indicates whether the search bar should grab
+      the focus when the page loads.
```

Comment:
Add "if not specified, it is set to false", or something similar?

Reply:
Done in 570e7d8.